src/collections/_documentation/meta/soc2.md

```diff
@@ -1,8 +1,5 @@
 ---
 title: 'SOC2 Report'
 sidebar_order: 2
+gatsby: true
```

Comment:
this will likely become the proposal for migrating pages now

- add gatsby: true
- delete content (makes it obvious its not used0